http/src/main/java/org/http4s/blaze/http/parser/BodyAndHeaderParser.java

```diff
@@ -386,6 +387,7 @@ else if (ch == HttpTokens.LF) {
                     final int buff_size = in.remaining();
 
                     if (remaining_chunk_size <= buff_size) {
+                        // End of this chunk
```

Comment:
😆 I was so disoriented by the non-FP code, it didn't even register with me that it was Java when I read it.